Week2/homework/maartjes_work.js

```diff
@@ -44,4 +42,22 @@ const tuesday = [
 
 const tasks = monday.concat(tuesday);
 
-// Add your code here
+const durationsInHours = tasks.map(task => Number(task.duration / 60).toFixed(2));
+
+const twoHrsLessDurations = durationsInHours.filter(twohrs => twohrs >= 2);
+
+const sallaryPerTask = durationsInHours.map(sallaryPerduration => sallaryPerduration * 20);
```

Comment:
Instead of `durationInHours` you should use the filtered durations from the previous statement.

The number 20 comes out of the blue without an explanation. Developers often call this a 'magic' number. I assume it is the hourly rate that Maartje earns. It is best to give it name so we can recognise it for what it is:

```js
const hourlyRate = 20;
```

Now use this variable instead of the number 20 in the mapping function.

Since what you are mapping are durations, is it again best to name the function argument `duration` rather than `sallaryPerduration`.

Note also the spelling error: `sallary` -> `salary`. The VSCode spell checker should have flagged this.